spring-cloud-modules/spring-cloud-open-telemetry/spring-cloud-open-telemetry1/src/main/java/com/baeldung/opentelemetry/api/client/PriceClient.java

```diff
@@ -0,0 +1,34 @@
+package com.baeldung.opentelemetry.api.client;
+
+import com.baeldung.opentelemetry.model.Price;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Component;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.client.RestTemplate;
+
+@Component
+public class PriceClient {
+
+    private static final Logger logger = LoggerFactory.getLogger(PriceClient.class);
```

Comment:
`logger` should be `LOGGER` if it's `private static final` (and I think it should be).

Reply:
I've made it change though not sure its convention.
Generally i've used as Logger as its not constant data rather just a final static object reference. https://www.slf4j.org/api/org/slf4j/Logger.html.